RockPaperScissors/RockPaperScissors/main.swift

```diff
@@ -6,5 +6,47 @@
 
 import Foundation
 
-print("Hello, World!")
+func inputMenu() {
+    print("가위(1), 바위(2), 보(3)! <종료 : 0>: ", terminator: "")
+    guard let inputNumber = readLine() else { return }
+    guard let userCard = Int(inputNumber) else { return }
+    
+    switch userCard {
+    case 1:
+        compareRandomNumber(userCard, with: createRandomNumber())
+        inputMenu()
+    case 2:
+        compareRandomNumber(userCard, with: createRandomNumber())
+        inputMenu()
+    case 3:
+        compareRandomNumber(userCard, with: createRandomNumber())
+        inputMenu()
+    case 0:
+        print("게임 종료")
+    default:
+        print("잘못된 입력입니다. 다시 시도해주세요.")
+        inputMenu()
+    }
+}
 
+func createRandomNumber() -> Int {
+    let randomNumber = Int.random(in: 1...3)
+    print(randomNumber)
+    return randomNumber
+}
+
+func compareRandomNumber(_ inputNumber: Int, with randomNumber: Int) {
```

Comment:
함수와 파라미터 이름을 정할 때, 중복되는 단어는 피하는 것이 좋다고 생각합니다.
그리고 현재 함수의 이름 `compareRandomNumber`는 무작위 숫자를 비교하는 기능을 할 것 같아요.

네이밍에 관한 내용은 프로젝트 참고 문서에 나와있는 공식문서 [Swift API Design Guidelines](https://swift.org/documentation/api-design-guidelines/)를 참고해주세요~!

완전히 좋은 네이밍은 아니니 참고로만 봐주세요!
위에서 `userCard`라는 이름을 사용하셨으니 이런 식으로 바꿔볼 수도 있겠네요 💭
```suggestion
func compareCard(user: Int, computer: Int) {
```

Reply:
compareRandomNumber는 compareCard로 
inputNumber, randomNumber는 user, computer로 변경하였습니다.